remixer_storage/src/main/java/com/google/android/libraries/remixer/storage/LocalStorage.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.libraries.remixer.storage;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import com.google.android.libraries.remixer.RemixerItem;
+import com.google.android.libraries.remixer.Variable;
+import com.google.android.libraries.remixer.serialization.GsonProvider;
+import com.google.android.libraries.remixer.serialization.StoredVariable;
+import com.google.android.libraries.remixer.sync.LocalValueSyncing;
+import com.google.gson.Gson;
+
+/**
+ * A {@link com.google.android.libraries.remixer.sync.SynchronizationMechanism} that stores values
+ * in a SharedPreferences object.
+ */
+public class LocalStorage extends LocalValueSyncing {
```

Comment:
If firebase as the source of truth ever becomes a thing, I would argue merging these two would be counterproductive. So far it's not bad enough to be worrisome, and it would make it significantly worse in the future, so I would ague against it.